### PR TITLE
[Breaking] - Change loaders names for consistency before release

### PR DESCRIPTION
**Pull request by magdalenafuentes**
This PR changes the names of the `TAU` loaders so they all have the same naming convention, and some minor fixes.

It is a breaking change as it changes loaders names, so better do this now.

### CODE REVIEW

**Comment by magdalenafuentes** [comment]
ccing @iranroman here as he might wanna take a look as well

File: docs/source/soundata.rst [hunk 1]
```diff
@@ -59,23 +59,23 @@ FSDnoisy18K
...
    :members:
    :inherited-members:
    
 
 TAU Spatial Sound Events 2019
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. automodule:: soundata.datasets.tau_sse_2019
+.. automodule:: soundata.datasets.tau2021sse_nigens
    :members:
```
Comment (justinsalamon) [inline docs/source/soundata.rst]: 2019 vs 2021 (title vs module)

File: scripts/make_tau_sse_2019_index.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import csv
 from pathlib import Path
 
-DATASET_INDEX_PATH = "../soundata/datasets/indexes/tau_sse_2019_index.json"
+DATASET_INDEX_PATH = "../soundata/datasets/indexes/tau2021sse_nigens_index.json"
```
Comment (justinsalamon) [inline scripts/make_tau_sse_2019_index.py]: 2019 vs 2021?

File: soundata/version.py [hunk 3]
```diff
@@ -2,4 +2,4 @@
 """Version info"""
 
 short_version = "0.1"
-version = "0.1.0-rc14"
+version = "0.1.0"
```
Comment (justinsalamon) [inline soundata/version.py]: This seems like a big PR. Let's first merge this one in as rc15m and then have a separate PR just for the official update to v0.1.0 and the release

**Review by justinsalamon — COMMENTED** [review]
Looks good mostly, some confusion (at least for me) on 2019 vs 2021 dataset, and let's separate this PR from the bump to 0.1.0 as a separate PR just for the formal release.

File: scripts/make_tau_sse_2019_index.py (hunk 2 above)
Comment (magdalenafuentes) [reply scripts/make_tau_sse_2019_index.py]: good catch, it got messed by the find and replace and this does not affect the tests

**Comment by iranroman** [comment]
looks good!

**Review by justinsalamon — APPROVED** [review]

**Pull request merged**